pages/common/esbuild.md

```diff
@@ -0,0 +1,36 @@
+# esbuild
+
+> An extremely fast JavaScript bundler and minifier.
+> More information: <https://esbuild.github.io/>.
+
+- Bundle a JavaScript code and print to stdout:
```

Comment:
```suggestion
- Bundle a JavaScript file and print to stdout:
```

Reply:
🤔 Bundles all the files into one based on the entry point.

Reply:
Changing to "file" doesn't change the meaning, IMO, and "JavaScript code" isn't perfect English ;)

Reply:
is `JavaScript application` better?

Reply:
Sure!
```suggestion
- Bundle a JavaScript application and print to stdout:
```